Sources/Purchasing/Configuration.swift

```diff
@@ -145,8 +153,21 @@ import Foundation
          * - Warning: This assumes your IAP implementation uses StoreKit 1.
          * Observer mode is not compatible with StoreKit 2.
          */
+        @available(*, deprecated, message: "Use with(purchasesAreCompletedBy:) instead.")
         @objc public func with(observerMode: Bool) -> Configuration.Builder {
-            self.observerMode = observerMode
+            self.purchasesAreCompletedBy = observerMode ? .myApp : .revenueCat
+            return self
+        }
+        /**
+         * Set `purchasesAreCompletedBy`.
+         * - Parameter purchasesAreCompletedBy: Set this to `.myApp` if you have your own IAP implementation and
+         * want to use only RevenueCat's backend. Default is `.revenueCat`.
```

Comment:
can we do double-backticks for these to auto-link to the symbols?

Reply:
Cool, I didn't known this feature!